src/transformer.ts

```diff
@@ -1,27 +1,14 @@
-import type { Transformer } from "./deps.deno.ts";
-
-const wellKnownParseModesMap = new Map([
-  ["html", "HTML"],
-  ["markdown", "Markdown"],
-  ["markdownv2", "MarkdownV2"],
-]);
+import type { ParseMode, Transformer } from "./deps.deno.ts";
 
 /**
  * Creates a new transformer for the given parse mode.
- * @param parseMode {string} The parse mode to use. If the parse mode is not in the well known parse modes map, it will be used as is.
- * @see https://core.telegram.org/bots/api#formatting-options for well known parse modes.
+ * @param parseMode {string} The parse mode to use.
+ * @see https://core.telegram.org/bots/api#formatting-options
+ *   for more information about formatting.
```

Comment:
TS does not support multi-line strings for @-tags in TSDoc.
```suggestion
 * @see https://core.telegram.org/bots/api#formatting-option for more information about formatting.
```

Reply:
Oh, didn't know... Now do!